OracleWebLogic/samples/12212-msiserver/container-scripts/launcher.sh

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/bash
```

Comment:
Can you please open a new issue for this, so that @mriccell can take a look? In the meantime, I will merge this PR.